src/redux/books/book.js

```diff
@@ -0,0 +1,24 @@
+// Actions
+const ADDBOOK = 'ADDBOOK';
+const REMOVEBOOK = 'REMOVEBOOK';
```

Comment:
Great implementation, just a slight suggestion since we are using a duck pattern our actions should have 'appName/reducerName/ACTIONTYPE'
- [ ] Let's kindly change our actions constants that would be all 😇